apps/playground/src/app/components/ag-grid/readonly-grid/readonly-grid.module.ts

```diff
@@ -21,4 +21,6 @@ import { ReadonlyGridComponent } from './readonly-grid.component';
     SkyDropdownModule,
   ],
 })
-export class ReadonlyGridModule {}
+export class ReadonlyGridModule {
+  public static routes = ReadonlyGridRoutingModule.routes;
```

Comment:
Putting the static `routes` property on the feature modules as well as the routing module?

Reply:
Yes. This is only needed if the routing module is in a different file from the feature module. We need the information to be on the feature module. However, the routes are often declared with the routing module and so we need to be able to grab them to put them on the feature module. I could have just exported a `const` but I opted for just putting them on a static property like we were doing elsewhere.